src/sentry/models/group.py

```diff
@@ -200,7 +200,7 @@ def by_qualified_short_id(self, organization_id: int, short_id: str):
 
     def by_qualified_short_id_bulk(self, organization_id: int, short_ids: List[str]):
         short_ids = [parse_short_id(short_id) for short_id in short_ids]
-        if not short_ids or any(short_id for short_id in short_ids if short_id is None):
+        if not short_ids or any(short_id is None for short_id in short_ids):
```

Comment:
I feel like I actually already fixed this (or something similar to it) in my original pr... wonder if I broke it on a rebase